### PR TITLE
Filipino Translation - 1 139 translated words . Partial translating recording-changes.asc

### PR DESCRIPTION
**Pull request by mind-blast**
Currently translating recording-changes.asc with total 1,139 translated words. This file still not finish. This is a partial translation.
Code text and other computer code text are not counted as translated words.

### CODE REVIEW

File: book/02-git-basics/sections/recording-changes.asc [hunk 1]
```diff
@@ -192,18 +192,18 @@ M  lib/simplegit.rb
 ?? LICENSE.txt
 ----
 
-New files that aren't tracked have a `??` next to them, new files that have been added to the staging area have an `A`, modified files have an `M` and so on.
-There are two columns to the output - the left-hand column indicates the status of the staging area and the right-hand column indicates the status of the working tree.
-So for example in that output, the `README` file is modified in the working directory but not yet staged, while the `lib/simplegit.rb` file is modified and staged.
-The `Rakefile` was modified, staged and then modified again, so there are changes to it that are both staged and unstaged.
+Bagong mga file na hindi sinusubaybayan ay merong isang `??` sunod sa kanila, ang bagong mga file na idinagdag sa pagtatanghal na lugar ay merong `A`, nabago na mga file ay merong `M` at iba pa.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/recording-changes.asc]: Ang mga bagong files...

File: book/02-git-basics/sections/recording-changes.asc [hunk 2]
```diff
@@ -192,18 +192,18 @@ M  lib/simplegit.rb
...
+Mayroong dalawang column sa output - ang kaliwang-kamay na column ay nagpapaliwanag sa katayuan sa pagtatanghal na lugar at ang kanang-kamay na column ay nagpapaliwanag sa katayuan sa tinatrabahong tree.
+Kaya halimbawa sa output na iyon, ang `README` na file ay binago sa tinatrabahong direktoryo pero hindi pa natanghal, habang ang `lib/simplegit.rb` na file ay binago at natanghal.
+Ang `Rakefile` ay binago, natanghal at pagkatapos ay binago muli, kaya may mga pagbabago sa mga ito na parehong natanghal at hindi natanghal.
 
 [[_ignoring]]
-==== Ignoring Files
+==== Hindi Pinapansin ang mga File
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/recording-changes.asc]: Pagbalewala ng mga File

File: book/02-git-basics/sections/recording-changes.asc [hunk 3]
```diff
@@ -30,14 +30,14 @@ Your branch is up-to-date with 'origin/master'.
...
 
-This means you have a clean working directory -- in other words, none of your tracked files are modified.
-Git also doesn't see any untracked files, or they would be listed here.
-Finally, the command tells you which branch you're on and informs you that it has not diverged from the same branch on the server.
-For now, that branch is always ``master'', which is the default; you won't worry about it here.
-<<_git_branching#_git_branching>> will go over branches and references in detail.
+Nangangahulugan ito na mayroon kang isang malinis na tinatrabahong direktoryo -- sa ibang salita, waka sa iyong mga sinusubaybayan na mga file ang nabago.
+Hindi rin makikita ni Git ang anumang hindi sinusubaybayan na mga file, o sila ay malista dito.
+Sa wakas, ang utos ay nagsasabi sa iyo na kung saan na branch ikaw ay nandoon at nagsasabi sayo na hindi ito naiiba mula sa parehong branch sa server.
+Para sa ngayonFor now, ang branch na yan ay laging ``master'', kung saan ay default; hindi ka mag-alala dito.
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/recording-changes.asc]: Para sa ngayon, ang branch

File: book/02-git-basics/sections/recording-changes.asc [hunk 4]
```diff
@@ -156,12 +156,12 @@ Changes not staged for commit:
 
 ----
 
-What the heck?
-Now `CONTRIBUTING.md` is listed as both staged _and_ unstaged.
-How is that possible?
-It turns out that Git stages a file exactly as it is when you run the `git add` command.
-If you commit now, the version of `CONTRIBUTING.md` as it was when you last ran the `git add` command is how it will go into the commit, not the version of the file as it looks in your working directory when you run `git commit`.
-If you modify a file after you run `git add`, you have to run `git add` again to stage the latest version of the file:
+Ano ang Ano?
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/recording-changes.asc]: Ano ba ‘yan?

File: book/02-git-basics/sections/recording-changes.asc [hunk 5]
```diff
@@ -156,12 +156,12 @@ Changes not staged for commit:
...
+Ngayon ang `CONTRIBUTING.md` ay nakalista bilang kapwa nakatanghal _at_ hindi nakatanghal.
+Paano ito posible?
```
Comment (severinolorillajr) [inline book/02-git-basics/sections/recording-changes.asc]: Paano ito naging posible?

**Comment by severinolorillajr** [comment]
Hi @robin-ho , I have checked your contributions.
Just need to update those items I have reviewed and we can merge this.
Thanks.

File: book/02-git-basics/sections/recording-changes.asc (hunk 1 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 2 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 3 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 4 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 5 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change.

**Comment by mind-blast** [comment]
Hi @severinolorillajr , i have already changed it . thank you.

**Pull request merged**